### ISSUE TITLE
Manage Telegram growth and discussion focus for March 2018

### ISSUE THREAD

**Issue by kitblake** [rchain/bounties#556]
Telegram admins have been round the clock active. The efforts to keep spammer/scammers at bay and the conversation focused on the tech, as opposed to the market, seems to be working.  \
The organic growth continues and needs continuous low-level management.  \
In the main group we're considering a weekly topic of discussion. ie. On-Chain Governance, Namespaces, Use Cases.
The main chat group is now at 1736 members.  A reasonable goal is 1750 members, aiming for 2k in April.  \
This issue supersedes #291.

Quoting @Mervyn853 in issue #291 (posted on Feb 4th, still not done):
Also, I would like to request that the social channels like telegram and reddit to be added to the main rchain.coop site as communication channels to assist us in working towards this goal. As often times people interested in the community are not able to find us through the main website.  \
@rchain_coop Telegram Chat Invite Link: https://t.me/rchain_coop  \
@patrick727  @pmoorman 

Measure of Completion:  See Combot statistics per chat group/channel.  \
Suggested Budget: $3720

This is the March activity for the rchain_coop group:  \
https://combot.org/chat/-1001123521235?start=01%2F03%2F2018&end=31%2F03%2F2018

**Comment by pmoorman** [comment]
_**TL;DR:** As a quickfix, I've added Telegram to the "join us" links in the website._

---

@Mervyn853 @kitblake regarding this...

> I would like to request that the **social channels like telegram and reddit to be added to the main rchain.coop site** as communication channels to assist us in working towards this goal.

I hear you, and this is a good idea. I have the keys to the website now, so we can roll out some updates. My schedule is to first update messaging (#522), team/about (#538) and update the information architecture (#555).

My next patch would probably be to update & improve the CTAs (Call to Actions) on the website, including those things. I'm putting that off for now, because I'd like to know my options with regards to email marketing, nurturing & list building first (see #219).  

**My best guess is that this will take another ~2-4 weeks.** 

To fix this for now, I've added Telegram to the "join us" links.

**Comment by kitblake** [comment]
Thanks for the tl;dr and quickfix @pmoorman.
